### DISCUSSION TITLE
Updating Wiki.js causes Lets-encrypt Error: Failed HTTP-01 Pre-Flight / Dry Run

### DISCUSSION THREAD

**Discussion by monchee** [requarks/wiki#3433] [Help / Questions]
**Question**
I was trying to update my instance of Wiki.js to the latest version, but fail to run on HTTPS. Defaulting to HTTP today for my presentation.

**Host Info:**
OS: Ubuntu 18.04, Docker
Wiki.js version: 2.5.117 from 2.4.107
Database engine: PostgreSQL 11
Guide followed: https://docs.requarks.io/install/upgrade + https://docs.requarks.io/install/ubuntu

I'm really inexperienced with docker but learning a lot. Many thanks everyone for the help.

**Comment by Smankusors** [comment]
how do you run Wiki.js on Docker?

**Comment by monchee** [comment]
I run it on Ubuntu 18.04 as per the installation instructions on the docs.

On Wed, 9 Sep 2020 at 1:57 am, Antony Kurniawan <notifications@github.com>
wrote:

>
>
> how do you run Wiki.js on Docker?
>
>
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Requarks/wiki/issues/2387#issuecomment-688973833>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEES5SMIFAHWLCS6NDX3JGTSEZH7DANCNFSM4Q64MENA>
> .
>
>
> --
Sent from Gmail Mobile
